### PR TITLE
Rename announcements to pinax.announcements

### LINKED ISSUE #31: Rename to pinax-announcements
In the [Groupware Starter Project](https://github.com/pinax/pinax-starter-projects/wiki/Groupware-Starter-Project) page milestone 1, this app is referenced as pinax-announcements and not django-announcements.

I think it's because it should be renamed.
I think the link should be updated until this app was renamed.

### PR DESCRIPTION
**Pull request by arthur-wsw**
Rename django-announcements to uses new pinax conventions.
Changelog:
- Drop support for django < 1.5
- Rewrite docs for readthedocs with markdown (template based on pinax-blog)
- Update test config (travis, tox, coveragerc, ...)
- Update .gitignore
- Generate django migrations
- Move templates to pinax/announcements/

Fixes #31

### CODE REVIEW

File: pinax/announcements/urls.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from django.conf.urls import patterns, url
+
+from pinax.announcements import views
+
+urlpatterns = patterns(
```
Comment (paltman) [inline pinax/announcements/urls.py]: `patterns(` is no longer supported.  Should just be a list of `url()` declarations.

**Comment by paltman** [comment]
Great stuff @arthur-wsw!  Thanks!

Just a few items of feedback:

1. Please see [comment on urls](https://github.com/pinax/django-announcements/pull/32#discussion-diff-53461027)
2. Can you resolve the Travis issue so that "All checks pass"?  I think it's a simple `flake8` issue.
3. Can you add in the [`makemigrations`](https://github.com/pinax/pinax-starter-app/blob/master/makemigrations.py) script from pinax-starter-app?  You'll need to make a couple modifications as this script is actually a template.  Let me know if you need some help.

Again, thanks for this awesome pull request!  Looking forward to seeing it merged.

**Comment by paltman** [comment]
@arthur-wsw I actually just merged #29 which addressed item 1 in my comment as well as the lint issue identified and mentioned in item 2.  This does mean that you'll need to update your branch with master.  There are [instructions for this here](https://help.github.com/articles/syncing-a-fork/).

**Comment by arthur-wsw** [comment]
I update the branch from pinax/master and add the makemigrations.py script from pinax-starter-app.

Let me now if more changes are needed for this pull request to be merge.

**Pull request merged**